### PR TITLE
Test case9

### PR DESCRIPTION
**Pull request by Kasia902**
@nbabii added 9th test case

### CODE REVIEW

File: pages/main.page.js [hunk 1]
```diff
@@ -26,6 +27,17 @@ class MainPage extends BasePage {
         })();
     }
 
+
+    async openFeedback() {
+        await allure.createStep("Open Feedback Page", async () => {
+            await this.getFeedbackLinkLocator().click();
+            await browser.getAllWindowHandles().then(function (handles) {
```
Comment (nbabii) [inline pages/main.page.js]: No need for `.then` method to use, please use only `await`

File: pages/main.page.js [hunk 2]
```diff
@@ -26,6 +27,17 @@ class MainPage extends BasePage {
...
+                browser.sleep(5000);
```
Comment (nbabii) [inline pages/main.page.js]: no sleeps please =)

File: pages/main.page.js [hunk 3]
```diff
@@ -26,6 +27,17 @@ class MainPage extends BasePage {
...
+                browser.switchTo().window(handles[1]);
```
Comment (nbabii) [inline pages/main.page.js]: use `await` in other case it will not wait for this action to be done

File: 09.spec.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+let MainPage = require("../pages/main.page");
```
Comment (nbabii) [inline 09.spec.js]: Looks like extra file

**Review by nbabii — CHANGES_REQUESTED** [review]

File: 09.spec.js (hunk 4 above)
Comment (Kasia902) [reply 09.spec.js]: I'm clicking on Feedback link from Main page, this file is required

Comment (nbabii) [reply 09.spec.js]: I mean `09.spec.js` there is same in specs folder

**Pull request merged**